crates/starknet-server/tests/get_transaction_by_hash.rs

```diff
@@ -133,10 +126,8 @@ mod get_transaction_by_hash_integration_tests {
             starknet_rs_core::types::DeclareTransaction::V2(declare_v2),
         ) = result
         {
-            assert_eq!(
-                declare_v2.transaction_hash,
-                FieldElement::from_hex_be(DECLARE_V2_TRANSACTION_HASH).unwrap()
-            );
+            let expected = "0x013070e9ff8b554254fb2d577ea75f0ed925947902893a23b8c738f8fbe5e122";
```

Comment:
Can we use const at the beginning of tests like it was before?

Reply:
These are very specific constants (like, the hash of a transaction), and highly unlikely that they will be used in more than one place. If in the future we really have the constant used in two places, I'm ok with extracting it. Also, it uses less lines like this and I think the clarity is the same.

Reply:
I'm aware that it will be not used elsewhere and it's not the point. Maybe I'll rephrase it. Currently, we have 2 syntaxes in tests one is the use of pub const that has names like `DECLARE_V1_TRANSACTION_HASH` defined at the top and you introduced new syntax like expected ="0x1" inside test.

Let's use one of it, `pub const` on the top or `expected = "0x1"`. I really don't care which one I just care about cohesion.

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/238/commits/f3aa8c1b805d592115c1a0f1fd8c92567cae80c8